pkgs/by-name/we/wemeet/package.nix

```diff
@@ -0,0 +1,251 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  autoPatchelfHook,
+  makeWrapper,
+  nss,
+  xorg,
+  desktop-file-utils,
+  libpulseaudio,
+  libgcrypt,
+  dbus,
+  systemd,
+  udev,
+  libGL,
+  fontconfig,
+  freetype,
+  openssl,
+  wayland,
+  libdrm,
+  harfbuzz,
+  openldap,
+  curl,
+  nghttp2,
+  libunwind,
+  alsa-lib,
+  libidn2,
+  rtmpdump,
+  libpsl,
+  libkrb5,
+  xkeyboard_config,
+  libsForQt5,
+  gcc,
+  pkg-config,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+  wireplumber,
+  libportal,
+  xdg-desktop-portal,
+  opencv,
```

Comment:
https://github.com/xuwd1/wemeet-wayland-screenshare/pull/6
Should we use the default OpenCV package instead of a pruned one like [opencv4WithoutCuda](https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=opencv)? IMO this is unnecessary since only core/imgproc are included.